lesson-15/index.html

```diff
@@ -0,0 +1,313 @@
+                    <button class="search-button col-md-12" id="searchHotel" type="submit" form="mainForm">Search</button>
+                </div>
+            </form>
+            <div class="apple-google">
+            <a href="https://play.google.com"><img class="google-img" src="image/header-sprite.svg#google" alt="google market"></a>
+            <a href="https://www.apple.com/ru/app-store/"><img class="apple-img" src="image/header-sprite.svg#app-store" alt="app-store"></a>
+            </div>
+        </article> 
+        <div class="filter-form hide display" id="filterForm">
+           <div class=counter-items-wrapper>
+                <div class="counter-item">
+                    <span class="counter-label">Adults</span>
+                    <div class="buttons-input-wrapper"> <input id="adultsInput" type="text" name="adults">
+                        <input disabled="disabled" type="button" class="counter-plus-minus-gray counter-btn" value="-" id="btnCounterAdultMinus" data-direction="minus">
+                        <p class="counter-current-value" id="counterAdult">0</p>
+                        <input type="button" class="counter-plus-minus-blue counter-btn" id="btnCounterAdultPlus" value="+" data-direction="plus">
+                    </div>
+                </div>
+                <div class="counter-item">
+                    <span class="counter-label">Children</span>
+                    <div class="buttons-input-wrapper"> <input id="childrenInput" type="text" name="children">
+                        <input disabled="disabled" type="button" class="counter-plus-minus-gray counter-btn-children" id="btnCounterChildrenMinus" value="-" data-direction="minus">
+                        <p class="counter-current-value-children" id="counterChildren">0</p>
+                        <input type="button" class="counter-plus-minus-blue counter-btn-children" id="btnCounterChildrenPlus" value="+" data-direction="plus">
+                    </div>
+                </div>
+                <div class="counter-item">
+                    <span class="counter-label">Rooms</span>
+                    <div class="buttons-input-wrapper">   <input id="roomsInput" type="text" name="rooms">                                        
+                        <input disabled="disabled" type="button" class="counter-plus-minus-gray counter-btn" id="btnCounterRoomsMinus" value="-" data-direction="minus">
+                        <p class="counter-current-value" type="text" id="counterRooms">0</p> 
+                        <input type="button" class="counter-plus-minus-blue counter-btn" value="+" id="btnCounterRoomsPlus" data-direction="plus">
+                    </div>
+                </div>
+                <div class="age-selector" id="childAgeQuastion"><p class="selector-text">What is the age of the child you’re<br>travelling with?</p></div></div>
```

Comment:
out of section
![image](https://user-images.githubusercontent.com/20189313/141701144-3ebe206f-52d0-4f86-b7dd-85cc372ae332.png)